Starfish/emulator/emulator.py

```diff
@@ -88,6 +90,10 @@ def __init__(
         self.eigenspectra = eigenspectra
         self.flux_mean = flux_mean
         self.flux_std = flux_std
+        self.factors = factors
+        self.factor_interpolator = LinearNDInterpolator(
```

Comment:
A linear interpolator is probably the right choice here for simplicity's sake, but I'm wondering if something like linear in log(factor) would be more accurate, given the Stefan-Boltzman law.

The problem is that this really depends on the specifics of the grid and what the natural units are. One solution would be to allow the user to pass in their own interpolator, if they really wanted to customize how this is done. But this is low priority, I'd say.

Reply:
something that should help a little is that `rescale=True`. This takes each input grid point and normalizes to unit cube (0 to 1). That should help a lot with interpolating between vastly different parameter scales like Teff and logg. I didn't see any serious changes in the diagnostic plots I was making between using the rescaling and not, but it seems like a good idea.

Reply:
Oh, perhaps I'm misunderstanding what `self.factors` are in this instance? I thought they were the normalization pre-factor scalars to convert a mean 1 spectrum back into fluxed or intensity units, and therefore would probably have a very large range if Teff even varies a little bit.

Reply:
factors here is 
`np.mean(fluxes, axis=1)` where `fluxes` are the raw fluxes from the spectral library. It sounds like that is what you are talking about, too, so it may be a concern. In my defense this is straight from Gully's code, so it was good enough for that work.